File: notifications/src/main/java/io/castled/notifications/trigger/models/SingleValueOperation.java
package io.castled.notifications.trigger.models;

import io.castled.notifications.trigger.enums.OperationType;
import io.castled.notifications.trigger.enums.PropertyType;

public class SingleValueOperation extends PropertyOperation {

    private String value;

    public SingleValueOperation(String value, OperationType operationType, PropertyType propertyType) {
        super(operationType, propertyType);
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: notifications/src/main/java/io/castled/notifications/trigger/models/SingleValueOperation.kt
package io.castled.notifications.trigger.models

import io.castled.notifications.trigger.enums.OperationType
import io.castled.notifications.trigger.enums.PropertyType

internal class SingleValueOperation(
    val value: String,
    operationType: OperationType,
    propertyType: PropertyType
) : PropertyOperation(operationType, propertyType)